libs/sdk-ui-kit/src/SettingWidget/StylingSettingWidget/StylingSettingWidget.tsx

```diff
@@ -147,14 +149,14 @@ const StylingSettingWidgetCore = <T extends StylingPickerItemContent>(
                     <FooterButtons>
                         <Button
                             className="gd-button-secondary"
-                            onClick={handleCancel}
-                            disabled={isApplyButtonDisabled}
+                            onClick={onCancel}
+                            disabled={disabledCancelButton ?? isApplyButtonDisabled}
                             value={intl.formatMessage({ id: "cancel" })}
                         />
                         <Button
                             className="gd-button-action"
                             onClick={handleApply}
-                            disabled={isApplyButtonDisabled}
+                            disabled={disabledApplyButton ?? isApplyButtonDisabled}
```

Comment:
It should be:
`disabled={disabledApplyButton !== undefined ? disabledApplyButton : isApplyButtonDisabled }`

Reply:
I fixed all comments. Please check again. Thanks